External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -3465,26 +3476,22 @@ private static ASExpr GetExpression(ScintillaControl sci, int position, bool ign
                         sb.Insert(0, "RegExp.#" + (subCount++) + "~");
                     }
                     // array access
+                    else if (c == ']') arrCount++;
```

Comment:
This change is strange... `if (c == '[')` can happen at any time but not the opposite? Why?